webblog/blog/migrations/0008_blog_owner.py

```diff
@@ -0,0 +1,21 @@
+# Generated by Django 2.2.5 on 2019-09-13 10:14
```

Comment:
This is your first Pull request. Remove all the migrations and replace them with just one migration

Reply:
done